profiles/views.py

```diff
@@ -20,12 +20,12 @@ class SavedCompaniesCreate(CreateAPIView):
     List of saved companies.
     Add a company to the saved list.
     """
-    permission_classes = [IsAuthenticated]
+    permission_classes = [IsAuthenticated, SelfCompany]
     serializer_class = SavedCompanySerializer
     pagination_class = ForumPagination
 
     def post(self, request):
-        user = request.user
+        user = self.request.user
```

Comment:
Why?

Reply:
Thank you for feedback. I don't remember why)) Delete 'self'